extensions/compose/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
strict would be prefered as it enable other checks as well

Reply:
updated